README.md

```diff
@@ -25,11 +25,14 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
+<<<<<<< HEAD
+=======
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
+>>>>>>> main
   <h3><b>Microverse README Template</b></h3>
 
 </div>
```

Comment:
Good job adding the README as requested by the previous reviewer. how ever, it seems you left your merge indicators on the file and kinda made it rough. please delete everything I pointed out there including the logo. The essence]of this is to give your project a professional look and let the header of your README look as shown below:
![image](https://user-images.githubusercontent.com/50990925/211779088-c416a84d-8ff4-48ea-9bff-458b2cf90408.png)